Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/check_mark_button.tsx

```diff
@@ -41,16 +34,10 @@ export class CheckMarkButton extends React.Component<Properties, {}> {
         alt='checkbox'
         height={this.props.size}
         width={this.props.size}
-        onClick={this.onClick}
+        onClick={this.props.onClick}
```

Comment:
This is good but it has a problem, if the user doesn't supply an ```onClick``` prop then this component will fail. To fix this review how to define ```defaultProps``` in TypeScript and then define a default prop for this that is a function that does nothing, basically ```() => {}```